docs/source/graphs.md

```diff
@@ -59,7 +59,7 @@ You can also save the output to a local `.graphql` file like so:
 rover graph fetch my-graph@my-variant > prod-schema.graphql
 ```
 
-> For more on passing values via `stdout`, see [Essential concepts](./essentials#using-stdout).
```

Comment:
should likely rename "essential concepts" to "conventions"

Reply:
Ahh good catch